src/robot/controller/localization_test.py

```diff
@@ -94,10 +95,9 @@ def test_odometry_position(localization_controller,
         """
         Test that move forward calls the gateway.
         """
-        asyncio.create_task(localization_controller.move_forward(10))
+        asyncio.create_task(localization_controller.move_forward(2 * math.pi))
```

Comment:
what?
Shouldn't it be rotate? since you're using PI

Reply:
Nah, test configuration is that wheel radius = 1, and that encoders emit 1 tick / rotation.

This means, 2pi moves the robot forward by 1 tick.